templates/Gemfile.erb

```diff
@@ -39,7 +39,7 @@ group :development, :test do
   gem "dotenv-rails"
   gem "factory_girl_rails"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.1.0"
+  gem "rspec-rails", "~> 3.3.0"
```

Comment:
I didn't get your "minor" mention. I was not aware that it works that way, I'm in favor of the change if that is the case.